src/main/java/org/gradle/profiler/studio/StudioGradleClient.java

```diff
@@ -113,7 +116,7 @@ public void close() {
     private void tryClose(Closeable closeable) {
         try {
             closeable.close();
-        } catch (IOException e) {
+        } catch (UncheckedIOException | IOException e) {
             e.printStackTrace();
         }
```

Comment:
This is getting way too complicated. Why do we need to catch an unchecked exception here? And if we want to do it properly, we should print its cause anyway. But why is this so complicated, I wonder?

Not that I know a better solution off the top of my head...

Reply:
BTW, I'm ok with merging this like it is, but it irks me that I have no better solution.

Reply:
fixed that magic with try-with-resources, thank you for the idea